packages/plugin-vue/src/helper.ts

```diff
@@ -2,9 +2,15 @@ export const EXPORT_HELPER_ID = 'plugin-vue:export-helper'
 
 export const helperCode = `
 export default (sfc, props) => {
+
+  if (typeof sfc === 'function' ) {
```

Comment:
**question:** May it improve it if we ask for `is not object`?
So e.g. `!sfc || typeof sfc !== 'object'`